sample-app/src/components/LocationBias.tsx

```diff
@@ -1,12 +1,9 @@
 import { useAnswersActions, useAnswersState, LocationBiasMethod } from '@yext/answers-headless-react';
+import { executeSearchWithUserLocation } from '../utils/geolocationutils';
 
 interface Props {
   isVertical: boolean,
-  geolocationOptions?: {
-    enableHighAccuracy?: boolean,
-    timeout?: number,
-    maximumAge?: number
-  },
+  geolocationOptions?: PositionOptions,
```

Comment:
Where is the `PositionOptions` type coming from? Are we missing an import or is this supposed to be defined in the file?

Reply:
yea, `PositionOptions` and `GeolocationPosition` are both built-in types